### PR TITLE
Sw 1290  avoid crashing on import failure

### PR DESCRIPTION
**Pull request by Yelko-mrbeam**
https://mr-beam.atlassian.net/browse/SW-1290

### CODE REVIEW

File: octoprint_mrbeam/model/empty_import.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, module, default_return=None):
+        super(EmptyImport, self).__init__()
+        self.default_return = default_return
+        self.error_message = 'Error occurred trying to import module {module}'.format(module=module)
+        self.log_error()
+
+    def log_error(self):
+        try:
+            from octoprint_mrbeam import mrb_logger
+            logger = mrb_logger("octoprint.plugins.mrbeam")
```
Comment (JFSolution) [inline octoprint_mrbeam/model/empty_import.py]: this should include a better description of the module that logs, just mrbeam is to wide

Comment (JFSolution) [reply octoprint_mrbeam/model/empty_import.py]: maybe something like octoprint.plugins.mrbeam.model.empty_import, or you coudl use the filename

**Comment by JFSolution** [comment]
Will the user see any error or info that docs is not working?

File: octoprint_mrbeam/model/empty_import.py (hunk 1 above)
Comment (Yelko-mrbeam) [reply octoprint_mrbeam/model/empty_import.py]: Nice catch 👏

Comment (Yelko-mrbeam) [reply octoprint_mrbeam/model/empty_import.py]: I added the string we would need to add on the import in order to use this class we could stick with this rule and actually maybe create a decorator to inject a logger to a class using this rule 🤔  shall we create a techdebt ticket for this? maybe @irlaec wants to take this as a first challenge 😋

**Review by JFSolution — APPROVED** [review]

**Pull request merged**